### DISCUSSION TITLE
Add --delete-empty-folders flag to index-assets console command

### DISCUSSION THREAD

**Discussion by robinbeatty** [craftcms/cms#13947] [Ideas] [answered]
We have an integration with Cloudinary DAM and Craft via [thomasvantuycom](https://github.com/thomasvantuycom)'s excellent plugin and need to periodically perform asset re-indexing via a cron with the console command. As the DAM potentially has a bunch of nested subfolders housing the assets, which can change, we need to be able to update the folders structures in craft too. The List Empty Folders For Deletion flag in the CP Index Assets page should do the job, but it's not currently available as a flag in the console command – can it be? (to delete missing folders without a further confirmation)

**Comment by brandonkelly** [comment, marked as answer]
Added for Craft 4.6 via #14026 🎉

**Reply by brandonkelly** [reply]
Craft 4.6.0 is out now with that change.
